openjdk/openjdk.mk

```diff
@@ -121,7 +121,7 @@ CUSTOM_NATIVE_OPTIONS :=
 
 ifneq ($(JDK_VERSION),8)
 	ifdef TESTIMAGE_PATH
-		ifneq ($(OS),OS/390)
+		ifeq ($(OS),OS/390)
```

Comment:
I believe this will make jdk native tests for other platforms fail.

Reply:
I agree with Sophia, the current change will break other platforms. If we have zos native now, we can simply set `JDK_NATIVE_OPTIONS` without if check.

Reply:
Got it. Thank you. I made JDK_NATIVE_OPTIONS common across platforms. could you please review now.